src/OpenSage.Game/Network/LobbyManager.cs

```diff
@@ -1,114 +1,141 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Linq;
 using System.Net;
-using System.Net.NetworkInformation;
+using System.Threading;
+using LiteNetLib;
+using LiteNetLib.Utils;
+using OpenSage.Network.Packets;
 
 namespace OpenSage.Network
 {
     public class LobbyManager
     {
-        public struct LobbyPlayer
+        private Game _game;
+        private EventBasedNetListener _listener;
+        private NetManager _manager;
+        private Thread _thread;
+        private bool _isRunning;
+        private NetPacketProcessor _processor;
+        private List<LobbyPlayer> _players = new List<LobbyPlayer>();
+
+        private static readonly NLog.Logger Logger = NLog.LogManager.GetCurrentClassLogger();
+
+        public IReadOnlyCollection<LobbyPlayer> Players => _players;
+        public string LocalAddress => NetUtils.GetLocalIp(LocalAddrType.IPv4);
```

Comment:
This is fine for now, but I think for some computers this will pick the wrong IP address if you have multiple network adapters, including virtual ones.